etc/README.hbs

```diff
@@ -55,20 +55,28 @@ Now you can install `kerberos` with the following:
 npm install kerberos
 ```
 
+#### Support Strategy
+
+There are two support tiers:
+
+- **Tier 1**: These platforms represent the majority of our users. Test failures on tier 1 platforms will block
+releases.
+- **Experimental**: May not compile, test suite may not exist or may not pass. Test failures on experimental platforms
```

Comment:
```suggestion
- **Experimental**: Test suite may not exist or may not pass. Test failures on experimental platforms
```

I do think our experimental platforms will always compile (we build a binary for these platforms) but they've never been tested

Reply:
Clearer, I like it.